swim/test_utils.go

```diff
@@ -206,6 +254,36 @@ func (c *swimCluster) Bootstrap() {
 	}
 }
 
+func (c *swimCluster) WaitForConvergence(timeout time.Duration) error {
```

Comment:
`// WaitForConvergence polls the checksums of each node in the cluster and returns when they all match, or until timeout occurs.`